pkg/cue/model/instance.go

```diff
@@ -32,20 +30,25 @@ import (
 
 // Instance defines Model Interface
 type Instance interface {
-	String() string
+	String() (string, error)
```

Comment:
does the string equals to the before one? Would you like to add some explanation about the difference?

Reply:
It's the same, we used to convert value to string first and store in the instance, now we store the value directly in instance and convert to string when `instance.String()` is called.